### DISCUSSION TITLE
Tiff images in Pimcore are loading without background

### DISCUSSION THREAD

**Discussion by manojthotla25** [pimcore/pimcore#10859] [Q&A / Support]
Hi Team,

We are facing issue with tiff format in Pimcore. Some of the tiff images are loading without background. Here some example where the tiff is loading without background. 

![image](https://user-images.githubusercontent.com/93549976/142430668-425c9116-7b66-4cb2-a20e-96ae2440f92e.png)

![image](https://user-images.githubusercontent.com/93549976/142430676-ab9becfd-6eee-4296-a8d7-bde44e6cd776.png)

Also when we download the same image the background is shown. Are we missing something. 

Can please provide some insight on the issue.

Thanks
Manoj

**Comment by dvesh3** [comment]
Hi @manojthotla25 Pimcore has a feature for auto-clipping asset thumbnails, if there clipping path defined on the assets. you can turn it off with config defined here https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_Clipping-Support
